org/ystia/docker/topologies/secured-docker-registry/types.yml

```diff
@@ -119,12 +120,13 @@ topology_template:
             default_instances: 1
     BlockStorage:
       type: tosca.nodes.BlockStorage
-      requirements:
```

Comment:
Well you are hitting one of the major differences between Alien and TOSCA [see `known differences` here](http://alien4cloud.github.io/#/documentation/2.1.0/devops_guide/dev_ops_guide.html) the attachment relation between Computes and BlockStorage are reversed.

Note that in Yorc we do follow the specification and we automatically reverse those relationships.
But to be imported into Alien we have to define those relationships as expected by Alien.
So please revert your modifications on attachements.